### PR TITLE
Q1 styling dialogs

### PR DESCRIPTION
**Pull request by ivannaranjo**
This PR splits the App Engine traffic management dialog into two dialogs. One dialog for specifying the traffic allocation and another one to manage the overall list of allocations.

The dialog to add the new allocation looks like this:
screen shot 2016-11-22 at 4 20 00 pm

The dialog to manage the overall list of allocations looks like this:
screen shot 2016-11-22 at 4 20 10 pm

### CODE REVIEW

File: GoogleCloudExtension/GoogleCloudExtension/AddTrafficSplit/AddTrafficSplitViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            _owner.Close();
+        }
+
+        private bool Validate()
+        {
+            int allocationValue = 0;
+            if (!Int32.TryParse(Allocation, out allocationValue))
+            {
+                UserPromptUtils.ErrorPrompt(
+                    message: $"Invalid value for allocation, must be a number: {Allocation}",
```
Comment (iantalarico) [inline GoogleCloudExtension/GoogleCloudExtension/AddTrafficSplit/AddTrafficSplitViewModel.cs]: Shouldn't these be resources?

File: GoogleCloudExtension/GoogleCloudExtension/AddTrafficSplit/AddTrafficSplitWindow.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    /// The dialog implementation.
+    /// </summary>
+    public class AddTrafficSplitWindow : CommonDialogWindowBase
+    {
+        /// <summary>
+        /// The view model for the dialog.
+        /// </summary>
+        private AddTrafficSplitViewModel ViewModel { get; }
+
+        private AddTrafficSplitWindow(IEnumerable<string> versions) : base("Add Traffic Split")
```
Comment (iantalarico) [inline GoogleCloudExtension/GoogleCloudExtension/AddTrafficSplit/AddTrafficSplitWindow.cs]: Shouldn't this be a resource?

File: GoogleCloudExtension/GoogleCloudExtension/SplitTrafficManagement/SplitTrafficWindowContent.xaml [hunk 3]
```diff
@@ -23,115 +24,64 @@
...
-                                 IsChecked="{Binding IsCookieChecked}" VerticalAlignment="Center" />
+                                 GroupName="ShardBy"
+                                 IsChecked="{Binding IsCookieChecked}"
+                                 Style="{StaticResource CommonRadioButton}"/>
                 </StackPanel>
-            </Border>
+            </GroupBox>
+            
+            <!-- The list of allocations. -->
+            <Label Content="_Allocations:"
```
Comment (iantalarico) [inline GoogleCloudExtension/GoogleCloudExtension/SplitTrafficManagement/SplitTrafficWindowContent.xaml]: Resource for content text

File: GoogleCloudExtension/GoogleCloudExtension/AddTrafficSplit/AddTrafficSplitWindow.cs (hunk 2 above)
Comment (ivannaranjo) [inline GoogleCloudExtension/GoogleCloudExtension/AddTrafficSplit/AddTrafficSplitWindow.cs]: Done.

**Pull request merged**